examples/get-markets/README.md

```diff
@@ -0,0 +1,25 @@
+## Create a Folder
+
+This use case demonstrates how the Spotify API can be utilized to create a new folder in your Spotify account.
```

Comment:
Here the example is about getting the available spotify markets, but the documentation say its about creating a folder (which is incorrect). so we either have to update the documentation or the code to be in sync.

btw I prefer having a more real example for this connector which includes 2-3 API calls in a meaningful way. We have added examples in the other connectors in the same way.